internal/resources/akscluster/schema.go

```diff
@@ -50,6 +50,17 @@ var ClusterSchema = map[string]*schema.Schema{
 		Default:     "default",
 		Optional:    true,
 	},
+	waitForHealthyKey: {
```

Comment:
Instead of `waitForHealthyKey` can you rename it as `waitForKubeConfig` ? and the description accordingly

Because here with bool flag we are actually waiting till pinniped extension is up and able to kubeconfig and not necessarily until all extensions are up and running